app/code/Transoft/Blog/Block/Bloglist.php

```diff
@@ -0,0 +1,143 @@
+<?php
+declare(strict_types=1);
+
+namespace Transoft\Blog\Block;
+
+use Magento\Catalog\Model\Locator\RegistryLocator;
+use Magento\Framework\Api\SearchCriteriaBuilder;
+use Magento\Framework\Api\SortOrderBuilder;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NotFoundException;
+use Magento\Framework\View\Element\Template;
+use Magento\Framework\View\Element\Template\Context;
+use Transoft\Blog\Api\Data\BlogInterface;
+use Transoft\Blog\Model\BlogRepository;
+
+/**
+ * Generates blog list
+ */
+class Bloglist extends Template
+{
+    /**
+     * ScopeConfigInterface $scopeInterface
+     */
+    private $scopeInterface;
+
+    /**
+     * @var RegistryLocator $registryLocator
+     */
+    private $registryLocator;
+
+    /**
+     * @var BlogRepository
+     */
+    private $blogRepository;
+
+    /**
+     * @var SortOrderBuilder $sortOrderBuilder
+     */
+    private $sortOrderBuilder;
+
+    /**
+     * @var SearchCriteriaBuilder $searchCriteriaBuilder
+     */
+    private $searchCriteriaBuilder;
+
+    /**
+     * Creates Bloglist block
+     *
+     * @param Context $context
+     * @param RegistryLocator $registryLocator
+     * @param ScopeConfigInterface $scopeInterface
+     * @param BlogRepository $blogRepository
+     * @param SortOrderBuilder $sortOrderBuilder
+     * @param SearchCriteriaBuilder $searchCriteriaBuilder
+     * @param array $data
+     */
+    public function __construct(
+        Context $context,
+        RegistryLocator $registryLocator,
+        ScopeConfigInterface $scopeInterface,
+        BlogRepository $blogRepository,
+        SortOrderBuilder $sortOrderBuilder,
+        SearchCriteriaBuilder $searchCriteriaBuilder,
+        array $data = []
+    ) {
+        $this->registryLocator = $registryLocator;
+        $this->scopeInterface = $scopeInterface;
```

Comment:
Remove field. Use $this->_scopeConfig (\Magento\Framework\View\Element\AbstractBlock::$_scopeConfig) instead